coreneuron/CMakeLists.txt

```diff
@@ -71,7 +62,7 @@ macro(mod2c_target name input)
     list(APPEND MOD2C_${name}_MODS "${mod2c_modname_}")
 
     add_custom_command(OUTPUT "${mod2c_output_}"
-      DEPENDS ${input} "${mod2c_install_prefix}/mod2c_core"
+      DEPENDS ${input} mod2c
```

Comment:
could you check if change in mod2c recompile mod files?

Reply:
It dont recompile when there is changes to mod2c